### PR TITLE
Expose JSObject and JSFunction

### PR DESCRIPTION
**Pull request by buffer**

### CODE REVIEW

File: tests/test_jsobject.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from v8py import JSObject
+
+def test_jsfunction(context):
```
Comment (tbodt) [inline tests/test_jsobject.py]: Pretty sure this wasn't supposed to be called `test_jsfunction`

File: tests/test_jsfunction.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from v8py import JSFunction
+
+def test_jsfunction(context):
+    f = context.eval('Math.sqrt')
+    assert isinstance(f, JSFunction)
```
Comment (tbodt) [inline tests/test_jsfunction.py]: These tests should probably go in test_function.py and test_object.py

**Comment by buffer** [comment]
The new patch takes care of the details you pointed out. Thanks, Angelo

**Comment by tbodt** [comment]
Should I push this to PyPI now, or wait for patches for triggering GC or setting the stack limit?

**Comment by buffer** [comment]
Hope I will be able to work on that stuff in the next hours. But if you are already planning a new release please do.

**Pull request merged**